README.md

```diff
@@ -128,6 +128,8 @@ Then, whenever the framework content is changed:
 - alternatively, run `npm run frontend-build:watch` to automatically rebuild the framework content whenever a framework YML file changes
 - generate schemas into the API's `json_schema` directory using the `scripts/generate-schemas.py` script in this repo.
 
+Alternatively, if you think `bower link` is more trouble than it's worth, you can just symlink the your project's `bower_components/digitalmarketplace-frameworks` to wherever this repo lives on your local filesystem and it's 99% as good as the above.
```

Comment:
`bower link` lets bower know what you're doing though, so it knows not to replace anything related to the dependency when you're running `bower update` after the version change. I'm pretty sure I had bower or one of our build steps rewrite my working directory when I was using similar manual approaches, so I'd be reluctant to suggest it unless you're confident it's safe.